Makefile-test.mk

```diff
@@ -103,24 +103,30 @@ test-multikueue-integration: gomod-download envtest ginkgo dep-crds kueuectl gin
 	$(PROJECT_DIR)/bin/ginkgo-top -i $(ARTIFACTS)/multikueue-integration.json > $(ARTIFACTS)/multikueue-integration-top.yaml
 
 CREATE_KIND_CLUSTER ?= true
+
+SINGLECLUSTER-E2E_TARGETS := $(addprefix run-test-e2e-singlecluster-,$(E2E_KIND_VERSION:kindest/node:v%=%))
+CUSTOMCONFIGS-E2E_TARGETS := $(addprefix run-test-customconfigs-e2e-,$(E2E_KIND_VERSION:kindest/node:v%=%))
 .PHONY: test-e2e
-test-e2e: kustomize ginkgo yq gomod-download dep-crds kueuectl ginkgo-top run-test-e2e-$(E2E_KIND_VERSION:kindest/node:v%=%)
+test-e2e: kustomize ginkgo yq gomod-download dep-crds kueuectl ginkgo-top $(SINGLECLUSTER-E2E_TARGETS) $(CUSTOMCONFIGS-E2E_TARGETS)
 
 .PHONY: test-multikueue-e2e
 test-multikueue-e2e: kustomize ginkgo yq gomod-download dep-crds ginkgo-top run-test-multikueue-e2e-$(E2E_KIND_VERSION:kindest/node:v%=%)
 
 .PHONY: test-tas-e2e
 test-tas-e2e: kustomize ginkgo yq gomod-download dep-crds kueuectl ginkgo-top run-test-tas-e2e-$(E2E_KIND_VERSION:kindest/node:v%=%)
 
+.PHONY: test-customconfigs-e2e
+test-customconfigs-e2e: kustomize ginkgo yq gomod-download dep-crds kueuectl ginkgo-top run-test-customconfigs-e2e-$(E2E_KIND_VERSION:kindest/node:v%=%)
```

Comment:
I think it does not need to depend on `kueuectl` (as `test-multikueue-e2e`) does not depend on it.